src/parse/value/css_function.rs

```diff
@@ -0,0 +1,395 @@
+use std::{borrow::Borrow, iter::Iterator};
+
+use codemap::Spanned;
+
+use crate::{
+    error::SassResult,
+    utils::{
+        as_hex, hex_char_for, is_name, peek_ident_no_interpolation, peek_until_closing_curly_brace,
+        peek_whitespace,
+    },
+    value::Value,
+    Token,
+};
+
+use super::super::Parser;
+
+impl<'a> Parser<'a> {
+    pub(super) fn eat_calc_args(&mut self, buf: &mut String) -> SassResult<()> {
+        buf.reserve(2);
```

Comment:
Shouldn't this be 3? `( )`, `(xxx)` (special case), `(#)`, `()` (2 but not average), `())`, `(x)` and most of them have 3 characters.

Reply:
I'm actually quite torn on this. I think it actually may be better to just remove this `reserve` call altogether.